besu/src/test/java/org/hyperledger/besu/PrivacyReorgTest.java

```diff
@@ -130,11 +141,15 @@ public class PrivacyReorgTest {
           .signAndBuild(KEY_PAIR);
 
   private final BlockDataGenerator gen = new BlockDataGenerator();
-  private BesuController besuController;
-  private PrivateStateRootResolver privateStateRootResolver;
   private PrivacyParameters privacyParameters;
   private Enclave mockEnclave;
   private Transaction privacyMarkerTransaction;
+  private final PrivacyReorgTestComponent context =
```

Comment:
before the variable was called `component`, should this be renamed for consistency